src/lib/widgets/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from './custom-widget/custom-widget.directive';
 export * from './layers-toggle/layers-toggle.component';
+export * from './layers-toggle/sub-layers-toggle.component';
```

Comment:
I'm not sure if it is necessary to export this here as this component is not going to be used outside of this library. If I'm wrong this can stay.

Reply:
I changed it, but it works either way and might be cleaner the first way.